wagtail/wagtailadmin/templates/wagtailadmin/base.html

```diff
@@ -37,3 +48,55 @@
         </div>
     </div>
 {% endblock %}
+
+{# Animation easter-egg: #}
+{% block extra_js %}
+    <script>
+        $(function() {
+          var sensitivity = 8; // the amount of times the user must stroke the wagtail to trigger the animation
+
+          var $logoContainer = $('.wagtail-logo-container__desktop');
+          var mouseX = 0;
+          var lastMouseX = 0;
+          var dir = '';
+          var lastDir = '';
+          var dirChangeCount = 0;
+
+          function enableWag() {
+            $logoContainer.removeClass('logo-serious').addClass('logo-playful');
+          }
+
+          function disableWag() {
+            $logoContainer.removeClass('logo-playful').addClass('logo-serious');
+          }
+
+          $logoContainer.mousemove(function(event) {
+            mouseX = event.pageX;
+
+            if (mouseX > lastMouseX) {
+              dir = 'r';
+            } else if (mouseX < lastMouseX) {
+              dir = 'l';
+            }
+
+            if (dir != lastDir && lastDir != '') {
+              dirChangeCount += 1;
+            }
+
+            if (dirChangeCount > sensitivity) {
+              enableWag();
+            }
+
+            lastMouseX = mouseX;
+            lastDir = dir;
+          })
+
+          $logoContainer.mouseleave(function() {
+            dirChangeCount = 0;
+            disableWag();
+          })
+
+          disableWag();
+        })
+    </script>
+{% endblock %}
```

Comment:
I'll sort this now. I think it makes sense to move the code to wagtailadmin/js/core.js, since that already contains code to initialise the menu, which is active on the same occasions that the logo is. (It also means we don't need to worry about people overriding the `{% block extra_js %}` and neglecting to insert `{{ block.super }}`...)